modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/storage_formatter.js

```diff
@@ -45,33 +45,42 @@ AUI.add(
 
 		StorageFormatter.prototype = {
 			formatStorage: function(size) {
-				var instance = this;
-
-				var suffix = instance.get(STR_SUFFIX_KB);
+				let formattedNumber;
```

Comment:
1. We cannot use `let` inside AUI module, use `var`.
1. Rename variable to `formattedSize`. Name should reflect variable meaning, not type.